lib/chrome/webdriver/chromeDelegate.js

```diff
@@ -49,6 +49,9 @@ class ChromeDelegate {
     // that can be fixed by emptying the logs twice :|
     await runner.getLogs(Type.PERFORMANCE);
     await runner.getLogs(Type.PERFORMANCE);
+    // Also make sure we clear the console log
+    // Hopefully one time is enough?
+    await runner.getLogs(Type.BROWSER);
```

Comment:
👍
This should be independent of skipHar, so maybe move it to the top of the function. I see there's similar issues with onStopIteration below.

Reply:
Ah good catch!

Reply:
yep let fix the other thing in another PR